tests/integrationv2/providers.py

```diff
@@ -178,12 +208,7 @@ def setup_server(self):
         if self.options.protocol == Protocols.TLS13:
             cmd_line.append('--tls13')
 
-        if self.options.protocol == Protocols.TLS13:
-            cmd_line.extend(['-c', 'test_all_tls13'])
-        elif self.options.protocol == Protocols.TLS12:
-            cmd_line.extend(['-c', 'test_all_tls12'])
-        else:
-            cmd_line.extend(['-c', 'test_all'])
+        cmd_line.extend(['-c', 'test_all'])
```

Comment:
`test_all` has TLS1.3 as well, was this intended to contain TLS1.3?

Reply:
On the server side we don't want to limit ourselves, otherwise the version negotiation tests will fail. So we allow all ciphers, and then check to see what was actually negotiated.